<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMessage</class>
 <widget class="QMainWindow" name="FormMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operator Message</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>581</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear messages history</string>
     </property>
     <property name="confirmAction">
      <bool>true</bool>
     </property>
     <property name="confirmText">
      <string>Do you want to clear operator messages history?</string>
     </property>
     <property name="program">
      <string>curl -X POST http://10.1.100.46:8080/clear-operator-msgs</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Note: This message will be displayed on the public machine status. Be careful what you type here.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSet">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Message</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>580</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QELabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>580</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="variable" stdset="0">
     <string>message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBeamReady">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Beam is ready for users </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInjectionLate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Injection will be late due to some technical issues</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBeamDelivery">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Beam delivery will be late due to some technical issues</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInjectionProgress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Injection to Storage-ring in progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMachineMaintenance">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine under maintenance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Operators on duty:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lstOperator1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M. Saleh (Tel.: 0791271613)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A. Abu Hussein (Tel.: 0770765069)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A. Al-Zoubi (Tel.: 0790968127)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M. Al-Natsheh (Tel.: 0796284121)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnSetOperators">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lstOperator2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M. Saleh (Tel.: 0791271613)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A. Abu Hussein (Tel.: 0770765069)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A. Al-Zoubi (Tel.: 0790968127)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M. Al-Natsheh (Tel.: 0796284121)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRemoveOperators">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
